plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergParquetFileWriter.java

```diff
@@ -40,6 +42,8 @@
     private final String outputPath;
```

Comment:
This should go to `ParquetFileWriter`

Reply:
Sorry I am not sure I follow this comment. I didn't put it here.